### DISCUSSION TITLE
part-ib/2015-46/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#175] [Question comments]
Part IB 2015 Markov Chains: Paper 1, Section II, H 

https://questions.tripos.org/part-ib/2015-46/

**Comment by lorcaheeney** [comment]
Which result do you use for the last part?

**Reply by ArthurConmy** [reply]
The way you do it is make a new Markov chain with the three states $(B, C), (E,D), F$ and then $e_1, e_2, e_3$ be the expected passage time from each of those three states.
